app/client/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" type="image/svg+xml" href="./white-anteater-logo-vectorized.svg"/>
+    <link rel="icon" type="image/svg+xml" href="./HackUCI_Logo.jpg"/>
```

Comment:
nit: I feel like we should follow the previous naming convention, so -> `hackuci-logo.jpg`  
But this isn't super important